config/pmd/ruleset.xml

```diff
@@ -9,6 +9,8 @@
   <rule ref="category/java/bestpractices.xml">
     <exclude name="GuardLogStatement"/>
     <exclude name="JUnitTestContainsTooManyAsserts"/>
+    <exclude name="LiteralsFirstInComparisons"/>
+    <exclude name="UnusedAssignment"/>
```

Comment:
I can't see the errors but can we not fix some of these exclusions?  Unused assignments seems like something easy to get rid of?

Reply:
Yes, I rid of it